recipes/hpv-kite/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Zatrzymaj przy błędzie
+set -o errexit
+
+# Budowanie projektu za pomocą Gradle
+sh ./gradlew clean build jar createDependenciesJar
```

Comment:
```suggestion
./gradlew clean build jar createDependenciesJar
./gradlew generateLicenseReport
```
We can run `generateLicenseReport` once we've used `sed` to add the `dependency-license-report` plugin.